### ISSUE TITLE
[feature request] OAuth Support

### ISSUE THREAD

**Issue by torrancew** [toabctl/jiracli#35]
Currently, jiracli wants a credential file to log in. In some settings, this can cause all sorts of annoying things, ranging from being unacceptable in the eyes of some policy, to causing forced logouts of browser sessions in the JIRA instance, which can make some workflows much more cumbersome. It would be awesome if OAuth was an option for those cases.

**Comment by toabctl** [comment]
Yeah. oauth support would be nice! But I'm currently busy and have not much time. But pull requests are of course welcome!

**Comment by JoseAlban** [comment]
+1
